package.json

```diff
@@ -8,6 +8,7 @@
   "license": "MIT",
   "dependencies": {
     "jest-environment-jsdom-thirteen": "^1.0.0",
+    "pretty-format": "^24.0.0",
```

Comment:
I remember when installing `jest`, it will automatically install `pretty-format` too. I'm not sure but would you please confirm it ? If `jest` installs for us, we don't need to add in our `dependencies`

Reply:
Well, maybe, but... But in some future `pretty-format` could be removed from our dependency (probably `jest`) and this project will get error because there is no `pretty-format` in `node_modules`. 

So this explicit dependency will prevent errors in such case.

Anyway, `node_modules` size is same. Even `yarn.lock` was not changed.

Only one problem: we have to update version of `pretty-format` too.

Reply:
So I am not sure it is good idea to have implicit dependency. If you force your opinion - sure I will remove it.

Reply:
I prefer having it as a dependency. Npm/Yarn are smart enough to install the package only once. Plus we will be notified by greenkeeper when the dependency updates and have more control in case pretty-format changes it API.